src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/ArmorForge.java

```diff
@@ -70,9 +70,9 @@ private boolean isCraftable(Inventory inv, ItemStack[] recipe) {
     }
 
     @ParametersAreNonnullByDefault
-    private void craft(Player p, ItemStack output, Inventory inv, Block dispenser) {
+    private void craft(Player p, ItemStack output, Inventory inv, Dispenser dispenser) {
         Inventory fakeInv = createVirtualInventory(inv);
-        Inventory outputInv = findOutputInventory(output, dispenser, inv, fakeInv);
+        Inventory outputInv = findOutputInventory(output, dispenser.getBlock(), inv, fakeInv);
```

Comment:
Same here, `#getBlock()` is a bit expensive and completely avoidable by just passing the `Block` object via a parameter